src/OneExplorer/OneStorage.ts

```diff
@@ -24,12 +25,157 @@ import {Logger} from '../Utils/Logger';
 import {ConfigObj} from './ConfigObject';
 import {Node, NodeType} from './OneExplorer';
 
-interface StringListMap {
-  [key: string]: string[];
+export {
+  BaseModelToCfgMap as _unit_test_BaseModelToCfgMap,
+  CfgToCfgObjMap as _unit_test_CfgToCfgObjMap,
```

Comment:
This is not test-related files but they are exported as `_unit_test_*`, why...?

Reply:
I see it :)
Just FYI, there are some workaround patterns such as https://stackoverflow.com/a/54116079 and https://stackoverflow.com/a/65422568.

Reply:
Yes, actually the workaround used in the links are the same to this - it's by aliasing its internal functions when exporting :-D